examples/timestamping/backend/tests/api.rs

```diff
@@ -43,18 +46,21 @@ const SERVICE_NAME: &str = "my-timestamping";
 
 fn init_testkit() -> (TestKit, MockTimeProvider) {
     let mock_provider = MockTimeProvider::new(SystemTime::now().into());
+    let time_service = TimeServiceFactory::with_provider(mock_provider.clone());
+    let timestamping = TimestampingService;
     let mut testkit = TestKitBuilder::validator()
-        .with_rust_service(
-            InstanceCollection::new(TimeServiceFactory::with_provider(mock_provider.clone()))
-                .with_instance(TIME_SERVICE_ID, TIME_SERVICE_NAME, ()),
-        )
-        .with_rust_service(InstanceCollection::new(TimestampingService).with_instance(
+        .with_artifact(time_service.get_artifact(), ())
```

Comment:
`get_instance` and `get_artifact` names is not so clear for me. `get_instance` for example, looks more like instance constructor but not like instance specification getter.

Reply:
Yeap I was puzzled with naming. I agree that for the instance it's more close to `create_` than `get_`, but for artifact semantically it just converts internal rust artifact to `ArtifactId`. `.get_instance()` -> `create_instance()` and left `.get_artifact()` as is?